src/Symfony/Bridge/Twig/Mime/WrappedTemplatedEmail.php

```diff
@@ -39,14 +39,16 @@ public function toName(): string
      *                                 some Twig namespace for email images (e.g. '@email/images/logo.png').
      * @param string|null $contentType The media type (i.e. MIME type) of the image file (e.g. 'image/png').
      *                                 Some email clients require this to display embedded images.
+     * @param string|null $name        A custom file name that overrides the original name (filepath) of the image
      */
-    public function image(string $image, ?string $contentType = null): string
+    public function image(string $image, ?string $contentType = null, ?string $name = null): string
     {
         $file = $this->twig->getLoader()->getSourceContext($image);
         $body = $file->getPath() ? new File($file->getPath()) : $file->getCode();
-        $this->message->addPart((new DataPart($body, $image, $contentType))->asInline());
+        $name = $name ?: $image;
```

Comment:
I guess this would be better?
```suggestion
        $name ??= $image;
```

Reply:
I wanted to prevent empty strings passed in `$name`, not only null/omitted parameters.